src/ui/profile_preferences.ui

```diff
@@ -84,7 +84,7 @@
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
-      <number>0</number>
```

Comment:
This change should not have been included as we should select the **first** tab on the preferences dialogue.  

:information_source:  As it happens there is specific code at the bottom of the `dlgProfilePreferences` constructor to override it because it gets miss-set like this by people not checking that they save from the `QDesigner` plugin/standalone utility with the first tab selected! :stuck_out_tongue_winking_eye:

Reply:
Yes but as you say, it does not matter anymore, so ...